code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -4,6 +4,8 @@
 // crit percent
 #define MALF_AI_ROLL_CRIT_CHANCE 5
 
+GLOBAL_VAR(turrets_upgraded) //If the turrets are upgraded
```

Comment:
are you sure there isn't a better place this could be scoped? Such as on the malf AI datum or something

Reply:
Moved to AI mob, where similar variables are stored (such as can_dominate_mechs)